docs/deploy/kuberay-with-MCAD.md

```diff
@@ -0,0 +1,95 @@
+# Kuberay integration with MCAD (Multi-Cluster-App-Dispatcher)
+
+The multi-cluster-app-dispatcher is a Kubernetes controller providing mechanisms for applications to manage batch jobs in a single or multi-cluster environment. For more details please refer [here](https://github.com/IBM/multi-cluster-app-dispatcher).
+
+## Use case
+
+MCAD allows you to deploy Ray cluster with a guarantee that sufficient resources are available in the cluster prior to actual pod creation in the Kubernetes cluster. It supports features such as:
+   
+- Integrates with upstream Kubernetes scheduling stack for features such co-scheduling, Packing on GPU dimension etc.
+- Ability to wrap any Kubernetes objects.
+- Increases control plane stability by JIT (Just-in Time) object creation.
+- Queuing with policies.
+- Quota management that goes across namespaces.
+- Dispatching jobs to any one of the Kubernetes clusters.
```

Comment:
Grammatically, it's slightly unclear what "the Kubernetes clusters" are.

For better context, perhaps it can be reworded like this:
`Support for multiple Kubernetes clusters; dispatching jobs to any one of a number of Kubernetes clusters.`

Reply:
Thanks, I will steal the edit and paste it to the docs :)